.gitignore

```diff
@@ -367,3 +367,15 @@ compose/local/django/pip-packages/*
 data/
 .envs/.local/.pypi-authfile
 
+# *.ttf and *.woff2 files
```

Comment:
This seems overly specifiy. Please use wildcards, e.g. static/js/*.ttf and static/js/*.woff2